browser/js/login/createUserAccount.html

```diff
@@ -0,0 +1,60 @@
+<div class="col-md-4 col-md-offset-4">
+	<div class="panel panel-default">
+		<div class="panel-heading">Create User</div>
+		<div class="panel-body">
+			<form name="createUserAccount" ng-submit="createUser(user)">
+				<div class="form-group">
+				  <label for="firstName">First Name</label>
+				  <input type="test"
+				  		required
+				  		name="firstName" 
+						class="form-control" 
+						id="firstName" 
+						placeholder="Enter First Name"
```

Comment:
Label and a placeholder may be a little busy.  Just something to consider, but I am okay with it either way.